test/run

```diff
@@ -182,7 +199,7 @@ def main():
 
     add_unit_tests(litani, test_dir, root, counter)
     add_e2e_tests(
-        litani, test_dir, counter, output_dir=output_dir)
+        litani, test_dir, counter, output_dir=output_dir, fast=args.fast)
```

Comment:
This line could become:

`litani, test_dir, counter, output_dir, args.fast)`

Reply:
Resolved